includes/payments/actions.php

```diff
@@ -70,12 +70,12 @@ function edd_complete_purchase( $order_id, $new_status, $old_status ) {
 					foreach ( $item->get_fees() as $key => $item_fee ) {
 						/** @var EDD\Orders\Order_Adjustment $item_fee */
 
-						$fee_id      = edd_get_order_adjustment_meta( $item_fee->id, 'fee_id', true );
 						$download_id = edd_get_order_adjustment_meta( $item_fee->id, 'download_id', true );
 						$price_id    = edd_get_order_adjustment_meta( $item_fee->id, 'price_id', true );
 						$no_tax      = edd_get_order_adjustment_meta( $item_fee->id, 'price_id', true );
+						// todo: should $no_tax be changed to (bool) 0.00 === $item_fee->tax;?
```

Comment:
Noted a potential issue on this `$no_tax` variable, also in `setup_cart_details` ... in `setup_fees`, the `$no_tax` variable is defined using:

```php
$no_tax   = (bool) 0.00 === $order_fee->tax;
```
and it seems these two should use the same? The `price_id` meta seems like a copy/paste error maybe?

Reply:
What you've described seems fine to me. I think you're right this looks like a copy paste error.